frontend/templates/troubleshooting/toc.tsx

```diff
@@ -144,6 +144,27 @@ export function MobileTOC({
    );
 }
 
+function useScrollToOnloadEffect() {
+   const { getItem } = useTOCContext();
+
+   const hash = window.location.hash;
+   const id = hash.replace('#', '');
+   const record = getItem(id);
+
+   const scrollTo = record?.scrollTo;
+   const [isMounted, setIsMounted] = useState(false);
+
+   useEffect(() => {
+      if (isMounted) {
+         return;
+      }
+
+      scrollTo && scrollTo();
+
+      setIsMounted(true);
+   }, [isMounted, scrollTo]);
```

Comment:
Pretty sure this'll work, because `useEffect` runs on mount, and passing no dependencies means we never re-run:
```suggestion
   useEffect(() => {
      scrollTo && scrollTo();
}, []);

Reply:
Oh, that's a good call. I believe that is the same. Let me double check and I'll commit if so